conduit/data/datasets/audio/ecoacoustics.py

```diff
@@ -117,9 +115,9 @@ def __init__(
         self.specgram_segment_len = specgram_segment_len
         self.resample_rate = resample_rate
         self._n_sgram_segments = int(self._AUDIO_LEN / specgram_segment_len)
-        self.hop_length = hop_length
-        self.num_freq_bins = num_freq_bins
-        self.preprocess_transform = T.Spectrogram(n_fft=num_freq_bins, hop_length=hop_length)
+        if transform is None:
+            transform = T.Spectrogram(n_fft=120, hop_length=60)
+        self.transform = transform
```

Comment:
needs setting here as used in preprocess

Reply:
maybe make `transform` a required argument? David seemed to think you want to hard-code the transform now.

Reply:
maybe rename it to `preprocessing` as well